Examples/Tests/electrostatic_sphere_eb/analysis_rz_mr.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Olga Shapoval, Edoardo Zoni
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+# This script tests the embedded boundary in RZ.
+# A cylindrical surface (r=0.1) has a fixed potential 1 V.
+# The outer surface has 0 V fixed.
+# Thus the analytical solution has the form:
+# phi(r) = A+B*log(r), Er(r) = -B/r.
+
+# Possible errors:
+# tolerance: 0.004
+# Possible running time: < 1 s
+
+import os
+import sys
+
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+tolerance = 0.0041
+
+fn = sys.argv[1]
+
+def find_first_non_zero_from_bottom_left(matrix):
+    nj = len(matrix[0,:])
+    ni = len(matrix[:,0])
+    for i in range(ni):
+        for j in range(nj):
+            if (matrix[i][j] != 0) and (matrix[i][j] != np.nan):
+                return (i, j)
+    return i, j
+
+def find_first_non_zero_from_upper_right(matrix):
+    nj = len(matrix[0,:])
+    ni = len(matrix[:,0])
+
+    for i in range(ni-1, -1, -1):
+        for j in range(nj-1, -1, -1):
+            if (matrix[i][j] != 0) and (matrix[i][j] != np.nan):
+                return (i, j)
+    return i,j
+
+def get_fields(ts, level):
+    er = []
+    phi = []
+    info = []
+    if level == 1:
+        er, info = ts.get_field('E_lvl1', 'r', iteration=0, plot=False)
+        phi, info = ts.get_field('phi_lvl1', iteration=0, plot=False)
+
+    elif level==0:
+        er, info = ts.get_field('E', 'r', iteration=0, plot=False)
+        phi, info = ts.get_field('phi', iteration=0, plot=False)
+
+    return er, phi, info
+
+def get_error_per_lev(ts,level):
+    er, phi, info = get_fields(ts, level)
+
+    nj = len(er[0,:])//2
+    dr = info.r[1] - info.r[0]
```

Comment:
`dr` can be accessed directly from `info`:
```suggestion
    dr = info.dr
```